tests/compute/test_heterograph.py

```diff
@@ -1741,7 +1741,7 @@ def test_format():
     # test_adj("int32")
     # test_inc()
     # test_view()
-    # test_view1("int32")
+    test_view1("int32")
```

Comment:
comment it?

Reply:
It doesn't matter since pytest just look for `test_xxx` functions.